File: benchmark/nodejs/arrsum.js
const arr = [];
for (let i = 0; i < 10000; i = i + 1) {
    arr.push(i);
}
let sum = 0;
const start = Date.now();
for (let i = 0; i < 10000; i = i + 1) {
    sum = sum + arr[i];
}
const end = Date.now();
console.log(`Sum: ${sum}`);
console.log(`Time: ${end - start}ms`);

File: benchmark/src/ruby/arrsum.rb
arr = []
for i in 0..9999
    arr.push(i)
end
sum = 0
start = Time.now.to_f * 1000
for i in 0..9999
    sum = sum + arr[i]
end
finish = Time.now.to_f * 1000
puts "Result: #{sum}"
puts "Time: #{(finish - start).round}ms"
